frontend/src/components/pages/overview/Overview.tsx

```diff
@@ -213,20 +212,23 @@ class Overview extends PageComponent {
                             </ul>
 
                             <ul className="resource-list">
-                                {news
-                                    ? news.map((x, i) => <li key={i}>
-                                        <a href={x.url} rel="noopener noreferrer" target="_blank" className="resource-link">
+                                <SkeletonText
+                                    isLoaded={Boolean(news)}
+                                    noOfLines={4}
```

Comment:
I see we had 3 lines previously, not sure if it's a big deal.

Reply:
I tried to check the UI when migrating and somewhere I tried to fine-tune some of the places. The original and this component are not exactly 1:1